### ISSUE TITLE
add config file support

### ISSUE THREAD

**Issue by hellcatz** [hellcatz/hminer#6]
It can be convenient when managing multiple machines to deploy a config file instead of command line options

**Comment by AbdulazizDXT** [comment]
Yes, sir, that would be amazing, and I am waiting impatiently to see the new update. Thank you for your interest in my proposal, and I am happy for that. Good luck, sir.

**Comment by CrXClarex** [comment]
Yes, it is preferable, and mostly 80% of people prefer the configuration file method and run the mining directly. I encountered difficulties previously in the command line, but I overcame them, but the configuration file is probably the best and most widespread in this period.

**Comment by gg-gtr** [comment]
nice thing

**Comment by kuwin** [comment]
hi ad... can you create a hminer program that runs on x64 arm or android phone?  that would be very good and convenient...thank you

**Comment by tradenet** [comment]
> hi ad... can you create a hminer program that runs on x64 arm or android phone? that would be very good and convenient...thank you

+1
